src/asset-swapper/utils/quote_report_generator.ts

```diff
@@ -107,6 +107,7 @@ export interface ExtendedQuoteReport {
     sourcesDelivered: ExtendedQuoteReportIndexedEntryOutbound[] | undefined;
     blockNumber: number | undefined;
     estimatedGas: string;
+    enableSlippageProtection?: boolean;
```

Comment:
[bit out of scope for this PR / optional] 
I think we can make `GetSwapQuoteParams.enableSlippageProtection` not optional (I think we only need to change `meta_transaction_service.ts`  in order to do that. 

Then we can make `ExtendedQuoteReport. enableSlippageProtection` not optional as well.

Reply:
I can create a separate task / PR for this change. It's small but it'll make the code a little cleaner.